vscode-dotnet-runtime-library/src/Acquisition/WinMacGlobalInstaller.ts

```diff
@@ -79,6 +80,9 @@ export class WinMacGlobalInstaller extends IGlobalInstaller {
                 if(conflictingVersion === this.installingVersion)
                 {
                     // The install already exists, we can just exit with Ok.
+                    this.acquisitionContext.eventStream.post(new DotnetAcquisitionAlreadyInstalled(getInstallKeyFromContext(this.acquisitionContext.acquisitionContext) ?? '',
```

Comment:
This provides very clear information about what to do if the version is installed. Does the user need to know that the version they required is not downloaded again?

Reply:
That's a good point -- it is already there, just in the OutputChannelObserver! :)

```ts
                    this.outputChannel.append(`${
                        (event as DotnetAcquisitionAlreadyInstalled).requestingExtensionId
                    } wants to install .NET ${
                        (event as DotnetAcquisitionAlreadyInstalled).installKey
                    } but it already exists. No downloads or changes were made.\n`);
```